app/src/main/java/com/ucmobiledevelopment/freeways/dao/IIncidentDAO.kt

```diff
@@ -2,17 +2,15 @@ package com.ucmobiledevelopment.freeways.dao
 
 import retrofit2.Call
 import retrofit2.http.GET
-import com.ucmobiledevelopment.freeways.dto.Incident
-import com.ucmobiledevelopment.freeways.dto.IncidentDTO
 import com.ucmobiledevelopment.freeways.dto.IncidentListDTO
 import retrofit2.http.Query
 
 interface IIncidentDAO {
 
     @GET("CrashAPI/crashes/GetCrashesByLocation?format=json")
-     fun getAllIncidents(
+    fun getAllIncidents(
         @Query("fromCaseYear") fromCaseYear: Int, @Query("toCaseYear") toCaseYear: Int,
-        @Query("state") state: Int, @Query("county") county: Int)
-        : Call<IncidentListDTO>
-
+        @Query("state") state: Int, @Query("county") county: Int
+    )
+            : Call<IncidentListDTO>
 }
```

Comment:
Reformated the code according to Kotlin. https://kotlinlang.org/docs/coding-conventions.html#formatting